service/src/main/resources/changelog/changesets/20230224_sarscov2_nextstrain_change_table.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: "change default data table name from 'nextstrain_mass_test' to 'nextstrain'"
```

Comment:
Is this a good ID?

Reply:
what should the ID be? the others all read like descriptions, too...